cmd/server/main.go

```diff
@@ -92,6 +94,9 @@ type ProxyRunOptions struct {
 	adminPort uint
 	// Port we listen for health connections on.
 	healthPort uint
+	// After a duration of this time if the server doesn't see any activity it
+	// pings the client to see if the transport is still alive.
+	keepaliveTime uint
```

Comment:
In this repo we've been using the `time.Duration` type for time related variables. (See https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/cmd/agent/main.go#L74). Can we change this to follow the same format?

Reply:
Thank you for pointing this out! It's a more clever way and simplify the other changes. I used `time.Duration` and updated the PR.